Core/Core/Inbox/View/InboxView.swift

```diff
@@ -91,13 +92,30 @@ public struct InboxView: View, ScreenViewTrackable {
                 } else {
                     archiveButton(message: message)
                 }
+                starButton(messageId: message.id, isStarred: message.isStarred)
             }
             .swipeActions(edge: .leading) {
                 readStatusToggleButton(message: message)
             }
         }
     }
 
+    @ViewBuilder
+    private func starButton(messageId: String, isStarred: Bool) -> some View {
+        Button {
+            model.starDidTap.send((!isStarred, messageId))
+        } label: {
+            let image = isStarred ? Image.starSolid : Image.starLine
+            let accessibilityLabel = String(localized: isStarred ? "Mark as Unstarred" : "Mark as Starred", bundle: .core)
```

Comment:
Just to doublecheck: will this localize correctly? If so then okay, if not please change to
```
isStarred ? String(...) : String(...)
```